src/test/java/Sysc4806Group/demo/entityTests/BookTest.java

```diff
@@ -0,0 +1,95 @@
+package Sysc4806Group.demo.entityTests;
+
+import Sysc4806Group.demo.entities.Book;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BookTest {
+
+    Book book;
+
+    @org.junit.jupiter.api.BeforeEach
+    void setUp() {
+        this.book = new Book( "123456", "XYZ Book", "Random Author", "Nelson", "google.com/images", 15);
+    }
+
+    @org.junit.jupiter.api.AfterEach
+    void tearDown() {
```

Comment:
What's this?

Reply:
> Add one/two non assertequals tests, just to add more tests to the bunch

Makes sense, will do that

Reply:
> What's this?

It doesn't do much right now so I can remove it but was hoping to use it later on